src/js/medium-editor.js

```diff
@@ -299,7 +299,9 @@ else if (typeof define === 'function' && define.amd) {
             }
             // Init toolbar
             if (addToolbar) {
-                this.initToolbar()
+                this.passInstance()
```

Comment:
:+1: I had tried to find other ways to get the instance of medium-editor passed to the extensions before the calls to `getButton()` were happening, but this is much cleaner (both calling `passInstance()` sooner, and adding an `init()` method into the extensions).  I look forward to using this stuff.